packages/components/pie-tag/test/visual/pie-tag.spec.ts

```diff
@@ -1,14 +1,69 @@
 
 import { test } from '@sand4rt/experimental-ct-web';
 import percySnapshot from '@percy/playwright';
-import { PieTag, TagProps } from '@/index';
+import type {
+    PropObject, WebComponentPropValues, WebComponentTestInput,
+} from '@justeattakeaway/pie-webc-testing/src/helpers/defs.ts';
+import {
+    getAllPropCombinations, splitCombinationsByPropertyValue,
+} from '@justeattakeaway/pie-webc-testing/src/helpers/get-all-prop-combos.ts';
+import {
+    createTestWebComponent,
+} from '@justeattakeaway/pie-webc-testing/src/helpers/rendering.ts';
+import {
+    WebComponentTestWrapper,
+} from '@justeattakeaway/pie-webc-testing/src/helpers/components/web-component-test-wrapper/WebComponentTestWrapper.ts';
+import { percyWidths } from '@justeattakeaway/pie-webc-testing/src/percy/breakpoints.ts';
+import { sizes, variants } from '@/defs';
 
-test.describe('PieTag - Visual tests`', () => {
-    test('should display the PieTag component successfully', async ({ page, mount }) => {
-        await mount(PieTag, {
-            props: {} as TagProps,
-        });
+// TODO: Currently setting the slot to use a straight up SVG
+//       This should be updated to use pie-icons-webc, but after some investigation, we think that we'll
+//       need to convert the webc icons to use Lit, as currently the components don't work well in a Node env like Playwright
```

Comment:
@dandel10n I appreciate this has already been merged but this comment isn't correct. The webc Icons now use Lit. So we likely should not be passing raw SVGs. 

Please could you take a look at using a `pie-icon-webc` component in this test if you have another PR to do?

Reply:
Sure, I will check and create a follow up PR 👍